Sources/ClientRuntime/Config/DefaultSDKRuntimeConfiguration.swift

```diff
@@ -119,7 +124,9 @@ public extension DefaultSDKRuntimeConfiguration {
     /// The retry strategy options to use when none is provided.
     ///
     /// Defaults to options with the defaults defined in `RetryStrategyOptions`.
-    static var defaultRetryStrategyOptions: RetryStrategyOptions { RetryStrategyOptions() }
+    static var defaultRetryStrategyOptions: RetryStrategyOptions {
+        RetryStrategyOptions(backoffStrategy: ExponentialBackoffStrategy())
+    }
```

Comment:
Since `RetryStrategyOptions` (in SmithyRetriesAPI) has been split from `ExponentialBackoffStrategy` (an implementation in SmithyRetries) the backoff strategy must be provided to the retry strategy options when it is created by the caller.